docs/certificates.md

```diff
@@ -0,0 +1,54 @@
+# Certificates
+
+We use the following certificates in production:
+
+- The certificate used for communication within EU, this needs to be signed by
+  the service provider.
```

Comment:
There are two EU certificates, the TLS certificate and the SEAL certificate.  And neither are signed by the service provider, they are signed by GlobalSign.